crates/matrix-sdk-crypto/src/store/mod.rs

```diff
@@ -1987,6 +1992,36 @@ impl Store {
         Ok(futures_util::stream::iter(sessions.into_iter().filter(predicate))
             .then(|session| async move { session.export().await }))
     }
+
+    /// Assemble a room key bundle for sharing encrypted history, as per
+    /// MSC4268.
```

Comment:
Can we please link to the MSC if we mention it. I have some scripts that find the PR just from `MSCXXXX` but other people might not have such advanced technology in their IDE.

Reply:
yup. Done in 89dee25.